### PR TITLE
ENG-10208 --force now deletes the contents of export overflow directory

### PR DESCRIPTION
**Pull request by manjujames**

### CODE REVIEW

File: src/frontend/org/voltdb/utils/VoltFile.java [hunk 1]
```diff
@@ -189,6 +189,22 @@ public static void resetSubrootForThisProcess() throws IOException {
...
+        }
+
+        if (deleteRoot) {
+            recursivelyDelete(file);
+        } else {
+            if (file.isDirectory()) {
+                for (File f : file.listFiles()) {
+                    recursivelyDelete(f);
+                }
+            }
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/VoltFile.java]: Shouldnt else delete the file?

File: src/frontend/org/voltdb/export/ExportManager.java [hunk 2]
```diff
@@ -388,6 +392,21 @@ private ExportManager(
         exportLog.info(String.format("Export is enabled and can overflow to %s.", cluster.getExportoverflow()));
     }
 
+    private void clearOverflowPath(CatalogContext catContext) throws ExportManager.SetupException {
```
Comment (akhanzode) [inline src/frontend/org/voltdb/export/ExportManager.java]: better name? clearOverFlowData() also log that you are doing this so we know when we look into log its was done.

File: src/frontend/org/voltdb/utils/VoltFile.java (hunk 1 above)
Comment (manjujames) [inline src/frontend/org/voltdb/utils/VoltFile.java]: No, I wrote this method for the cases when DONT want to delete the root directory, but just its contents.

**Comment by manjujames** [comment]
Squash merged.

**Pull request closed without merging**